src/libraries/illink-sharedframework.targets

```diff
@@ -66,6 +56,49 @@
       <ILLinkArgs>$(ILLinkArgs) -r @(_AssembliesToLink->'%(Identity)', ' -r ')</ILLinkArgs>
     </PropertyGroup>
 
+    <!-- pass suppressions xml files -->
+
+    <PropertyGroup>
+      <ProjectILLinkSuppressionsFile>src\ILLink\ILLink.Suppressions</ProjectILLinkSuppressionsFile>
+      <ProjectILLinkSuppressionsMonoSpecificFile>$(ProjectILLinkSuppressionsFile).Mono</ProjectILLinkSuppressionsMonoSpecificFile>
+      <ProjectILLinkSuppressionsWasmSpecificFile>$(ProjectILLinkSuppressionsFile).wasm</ProjectILLinkSuppressionsWasmSpecificFile>
+      <CoreLibSharedILLinkSuppressionsFilesPrefix>$(CoreLibSharedDir)ILLink\ILLink.Suppressions.Shared</CoreLibSharedILLinkSuppressionsFilesPrefix>
+    </PropertyGroup>
+
+    <!-- Collate XML files not bin-placed in earlier per-library linker run -->
+    <ItemGroup>
+      <!-- CoreLib doesn't use bin-place logic -->
+      <_SuppressionsXmls Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).xml" />
+      <_SuppressionsXmls Condition="'$(TargetOs)' != 'Windows_NT'" Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).NonWindows.xml" />
+      <_SuppressionsXmls Condition="'$(RuntimeFlavor)' == 'Mono'" Include="$(MonoProjectRoot)netcore\System.Private.CoreLib\$(ProjectILLinkSuppressionsFile).xml" />
+
+      <!-- Assemblies that were not included in per-library linker run -->
```

Comment:
Why aren't these included in the per-library linker run?

Reply:
I've addressed in a new commit. Only CoreLib xml files are manually bin-placed now (they have to be AFAIK).